source/content/guides/wordpress-composer/04-multisite-configuration.md

```diff
@@ -0,0 +1,78 @@
+
+</Alert>
+
+## Set Up WordPress Multisite
+
+1. Set the site's connection mode to Git:
+
+    ```bash{promptUser: user}
+    terminus connection:set <site>.dev git
+    ```
+2. Open the `config/application.php` file in your site's codebase and add the following line:
+
+	```php
+	Config::define( 'WP_ALLOW_MULTISITE', true );
+	```
+3. Visit your development site's WordPress admin and navigate to **Tools** > **Network Setup**.
+    ![Network Setup page](../../../images/wordpress-composer/04-multisite-network-setup.png)
+4. Choose **Sub-domains** or **Sub-directories** (depending on your needs), fill in the Network Title and Admin Email, and click **Install**.
+5. You will be given instructions to update your `config/application.php` file. Below where you added the `WP_ALLOW_MULTISITE` line, copy and paste the code provided. The `SUBDOMAIN_INSTALL` value will be `true` or `false` depending on the option you chose in the previous step. For example, if you chose subdirectories, your `config/application.php` file should look like this:
+
+	```php
+	Config::define( 'MULTISITE', true );
+	Config::define( 'SUBDOMAIN_INSTALL', false );
+	// Use PANTHEON_HOSTNAME if in a Pantheon environment, otherwise use HTTP_HOST.
+	Config::define( 'DOMAIN_CURRENT_SITE', defined( 'PANTHEON_HOSTNAME' ) ? PANTHEON_HOSTNAME : $_SERVER['HTTP_HOST'] );
+	Config::define( 'PATH_CURRENT_SITE', '/' );
+	Config::define( 'SITE_ID_CURRENT_SITE', 1 );
+	Config::define( 'BLOG_ID_CURRENT_SITE', 1 );
+	```
+	![Enabling the Network](../../../images/wordpress-composer/04-multisite-enable-network.png)
+6. Save your changes and commit them to your Git repository.
+7. Log into your site. When you log back into your WordPress admin dashboard, you will see a new **My Sites** menu item in the top left corner. This is where you can manage your network of sites. You now have a WordPress subdirectory multisite network set up on a WordPress (Composer Managed)-based upstream.
+	![My Sites](../../../images/wordpress-composer/04-multisite-my-sites.png)
+	
+## Using Subdomain Multisite on Pantheon
+
+Pantheon does not support sub-subdomains on the `pantheonsite.io` platform domain. This means that sub-sites on a subdomain multisite will not be accessible by default. See the [Mapping Custom Hostnames](/guides/multisite/config/#map-custom-hostnames-subdomain-configurations-only) panel in the Multisite Configuration guide for how to set up a custom domain for your sub-sites.
+
+Alternately, you can use [Subdomain to Subdirectory Multisite Conversion](/guides/multisite/search-replace/#subdomain-to-subdirectory-multisite-conversion) to make your dev and test environments use subdirectory multisite installs while your live site uses subdomains. In this case, your `SUBDOMAIN_INSTALL` value would be conditional based on the `$_ENV['PANTHEON_ENVIRONMENT']` value as documented in the Multisite Search and Replace guide.
```

Comment:
@pwtyler @scottbuscemi Can one of you confirm that this is accurate for WP Composer?

Reply:
Answered in [Jira](https://getpantheon.atlassian.net/browse/SITE-1134?focusedCommentId=234695) -- WPMS search-replace currently does not support composer-based sites. Updated to document workaround in https://github.com/pantheon-systems/documentation/pull/9173/commits/11a3e1d2dd27a1c2750fa557c47080c24a3b71b7